.github/workflows/dev-ecr-deploy.yml

```diff
@@ -0,0 +1,53 @@
+name: Build & Tag Container, Push to ECR, Deploy to Dev
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build, Tag & push to ECR, Deploy task
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+
+      - name: Install Dependencies
+        run: yarn install
+
+      - name: Build
+        run: |
+          yarn clean
+          yarn build
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_CI_USER_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_CI_USER_SECRET_ACCESS_KEY }}
+          aws-region: us-east-2
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Build, tag, and push image to Amazon ECR
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: optimism/rollup-full-node
+          IMAGE_TAG: latest
+        run: |
+          cd packages/rollup-full-node
+          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+
+      - name: Stop existing dev-rollup-full-node ECS task
+        run: |
+          ./.github/scripts/stop-ecs-task.sh dev-rollup-full-node rollup-full-node
```

Comment:
I am a little confused why `Stop existing dev-rollup-full-node ECS task` is after `Build, tag, and push image to Amazon ECR` as opposed to before. Where is it starting up the new container?

Other than this everything in the config makes a lot of sense!!! Very cool!!

Reply:
The `rollup-full-node` service in the `dev-rollup-full-node` cluster  aims to always have exactly 1 healthy task that is the tagged `latest` image in ECR. So we push the new image so that it becomes the tagged `latest` version in ECR and then kill the currently-running task so that a new one with the new image gets auto-started in its place.

Reply:
We'll probably need a better approach for higher environments, but this should work for now for dev.